src/node_dotenv.cc

```diff
@@ -101,35 +92,135 @@ Local<Object> Dotenv::ToObject(Environment* env) {
   return result;
 }
 
-void Dotenv::ParseContent(const std::string_view content) {
-  std::string lines = std::string(content);
-  lines = std::regex_replace(lines, std::regex("\r\n?"), "\n");
+std::string_view trim_spaces(std::string_view input) {
+  if (input.empty()) return "";
+  if (input.front() == ' ') {
+    input.remove_prefix(input.find_first_not_of(' '));
+  }
+  if (!input.empty() && input.back() == ' ') {
+    input = input.substr(0, input.find_last_not_of(' ') + 1);
+  }
+  return input;
+}
 
-  std::smatch match;
-  while (std::regex_search(lines, match, LINE)) {
-    const std::string key = match[1].str();
+void Dotenv::ParseContent(const std::string_view input) {
+  std::string lines =
+      std::regex_replace(std::string(input), std::regex("\r\n?"), "\n");
```

Comment:
This temporary code checks if it fixes the unit test issues on Windows.
If it works I'll push a fix without using `regex_replace`